### ISSUE TITLE
[Feature request] Minimum and maximum width/height on paddings

### ISSUE THREAD

**Issue by jackiecorn** [DWilliames/paddy2-beta#4]
One feature I loved about V1 is setting minimum or maximum width/height on the background layer. Would really like to see this in V2. Thanks!

**Comment by DWilliames** [comment]
Got it in the road map. 👍 
Curious; would you expect that to restrict the other content to remain within the 'padding layer' when it hit's it max/min sizes? Or would the content not be affected; only the 'padding layer' would be restricted?

**Comment by jackiecorn** [comment]
I honestly think the content should be restricted to the hard rules of max/min sizes, however I could see if you set max/min on both width and height, at some point the content will have nowhere to go. It's a tough challenge to solve. But even having only the padding layer be restricted will be great!

**Comment by trvsss** [comment]
I agree on it restricting content. The use case that comes to mind for me is assigning a min or max in use for layout within a screen. This means you would want a sidebar for example to be a min-width and it would re-layout everything inside as the width changes.

**Comment by DWilliames** [comment]
Hmm 🤔 Is there a way to handle both cases – maybe taking into account Sketch's resizing constraints?

Maybe it is just a hard rule for sizing the padding layer itself. But if you have set a 'fixed width' / 'fixed height' on a group; it will respect that after resizing/moving layers or adjusting padding layers; and re-layout the layers so that it stays the same width/height.

Thoughts?

**Comment by jackiecorn** [comment]
I'm thinking more about the case where when you edit the text inside a fixed size group. What will happen to the text content? 
I'd imagine because width is fixed, so the text has to wrap, but that would cause it to have an increased height, which may go beyond the fixed height of the group.
But I guess there's no way to accommodate both without having the content overflow. Even css does that.
[https://codepen.io/anon/pen/JvpevY](https://codepen.io/anon/pen/JvpevY)

This wouldn't be a problem with min width/height since the content size is always larger than that before the rule kicks in.
For max width/height, I think when the content size goes beyond the max size, the content should be untouched, and the padding layer should stay at the max size, similar to the CSS behavior.
